src/researchhub_case/views/external_author_claim_case_view.py

```diff
@@ -74,27 +76,53 @@ def create(self, request, *args, **kwargs):
         detail=False,
         methods=[POST],
     )
-    def search_scholarly_by_name(self, request):
+    def search_scholarly_profiles(self, request):
+        """
```

Comment:
Is this file even the right place for such logic? Let me know if there's abetter place besides the view.